python/openvino_tokenizers/tokenizer_pipeline.py

```diff
@@ -839,18 +839,46 @@ def get_ov_subgraph(self, input_nodes):
 
         op_inputs = []
         input_nodes_iter = iter(input_nodes)
-        for node in self.inputs:
-            if isinstance(node, Sequence):
-                op_inputs.extend(islice(input_nodes_iter, 3))
-            elif isinstance(node, AddToken):
-                # Put a scalar as a ragged tensor with scalar shape and a single element
+        from itertools import groupby
+        
+        # connect state flag to end_outputs
+        import openvino as ov
+        from openvino.runtime.op import Constant
+        from openvino.runtime.opset13 import assign, read_value, multiply
+        from openvino.runtime.op.util import VariableInfo, Variable
```

Comment:
I think we should move these imports to the top and merge them with existing ones.

Reply:
this was a leftover code which is now at utills.py, moved this imports to the beginning of utils.py and merged